docs/pages/data/explore/tableau.md

```diff
@@ -116,10 +134,26 @@ Start or restart _Tableau Desktop_ for the JDBC Driver installation to take effe
     - **Dialect** = `SQL92`
     - **Username** = `<Client ID>`, where a client id value is [generated with your application token](https://awesome-astra.github.io/docs/pages/astra/create-token/).
     - **Password** = `<Client Secret>`, where a client secret is [generated with your application token](https://awesome-astra.github.io/docs/pages/astra/create-token/). 
+
 - Click the _Sign In_ button to establish a connection.
 
 <br/><img src="https://awesome-astra.github.io/docs/img/tableau/connection-dialog.png" /><br/>
 
+**Ing driver:**
+
+- Select _Other Databases (JDBC)_ under _Connect_
+- Fill out the dialog box with the connection information:
+
+```
```

Comment:
Should we add which fields in the Other Databases (JDBC) form map to these entries? 

For example
URL:
Dialect:
Username:
Password:
Property File: